### DISCUSSION TITLE
Can't start locally on Windows 11

### DISCUSSION THREAD

**Discussion by mbbentz** [twilio-professional-services/flex-project-template#107] [Q&A] [answered]
Hey found this template and think it's a great idea! I cloned the repo and followed the steps outlined in the "setup guide" portion in the main readme. Ultimately, I can't get the plugin/functions to start locally.

During setup, `npm ci` originally failed with error **EBADPLATFORM** thrown by fsevents... I got around this by changing the `package-lock.json` file `os` entry for `fsevents` to `win32`.  This allowed me to get through installation. It prompted me for my Twilio Auth Token and I provided it.

When I try to run `npm run start:local:v2` it stalls at `Starting development server (static)...`. It opens my default browser and tries navigating to `localhost:3000`, but it just spins. I've left it for up to 10 minutes and it just spins. It never errors out or succeeds.

**Comment by mbbentz** [comment]
I'll add that I don't think it has anything to do with `serverless functions`. I'm able to start that independently without any issue. If I then try to manually launch the plugin using `npm run start:plugin:v2`, it stalls at `Starting development server (static)...` and the browser tab just spins.

**Comment by jared-hunter** [comment]
Hi @mbbentz 

Thank you for your feedback, very much appreciated.

I'll be tidying up the minor item there with the package-lock OS version but taking a look at the main issue....

it seems the reason its spinning is the local app wont properly start, theres an issue with the webpack that seems to be unrelated to this plugin specifically.  

I created a fresh plugin from start using "twilio flex:plugins:create MySamplePlugin" and then "npm install" and "twilio flex:plugins:start" - i get the same issue, we'll need to see whats going on with the local launcher on windows 

please let me know if you are seeing the same thing

<img width="765" alt="Screenshot 2022-12-05 at 10 09 38" src="https://user-images.githubusercontent.com/44647761/205686073-1ef2b3c8-6e4e-4a24-956c-715e2822fb78.png">

**Reply by mbbentz** [reply]
I did see that same error yesterday when trying to start the v2 plugin directly within it's directory. I figured it was just a product of trying to take that shortcut. I didn't see that error when trying to launch the template using `npm run start:local:v2`... but maybe it was still the cause, just failing silently.

**Comment by jared-hunter** [comment, marked as answer]
Hey @mbbentz 

Following up on this a little more.  I raised an [issue](https://github.com/twilio/flex-plugin-builder/issues/759) on the plugin builder 

Also, using NVM for windows (guide [here](https://linuxhint.com/downgrade-node-version-windows/)) - i tested the latest major versions of node. 

- 19 - same issue
- 18 - same issue
- 17 - same issue
- 16 - success 
- 15 - success 
- 14 - success 

it looks like windows has an issue starting the process properly when running it through a wrapper process, whether that is in shell.js or concurrently.  I switched the scripts to use npm-run-all so you'll need to take a sync or a fresh clone of the template for start:local:v2 to work on windows.
